tests/AppBundle/Controller/AdherentControllerTest.php

```diff
@@ -207,6 +207,50 @@ public function testEditAdherentInterests()
         }
     }
 
+    public function testAdherentChangePassword()
+    {
+        $this->authenticateAsAdherent($this->client, 'carl999@example.fr', 'secret!12345');
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/espace-adherent/mon-profil/changer-mot-de-passe');
+
+        $this->assertCount(1, $crawler->filter('input[name="adherent_change_password[old_password]"]'));
+        $this->assertCount(1, $crawler->filter('input[name="adherent_change_password[password][first]"]'));
+        $this->assertCount(1, $crawler->filter('input[name="adherent_change_password[password][second]"]'));
+
+        // Submit the profile form with invalid data
```

Comment:
Please can you add for this step (invalid data) an assert to be sure that the password have not been updated? With `$this->authenticateAsAdherent')` or directly from the database?